src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -673,28 +666,26 @@ protected override StatementSyntax CreateAssignmentExpressionStatement(SyntaxTok
                     .WithUsingKeyword(usingKeyword);
             }
 
-            protected override async Task<GeneratedCode> CreateGeneratedCodeAsync(OperationStatus status, SemanticDocument newDocument, CancellationToken cancellationToken)
+            protected override async Task<GeneratedCode> CreateGeneratedCodeAsync(
```

Comment:
actual generation of code no long involves 'status' as there's nothing to report once we've decided to generte.